### ISSUE TITLE
nv_core.h not found

### ISSUE THREAD

**Issue by kir12** [nya3jp/python-animeface#3]
I was attempting to set up a new python environment for animeface, but installation using `pip install animeface` fails with the following error:

```
animeface/_nvxs.c:21:10: fatal error: nv_core.h: No such file or directory
     #include "nv_core.h"
              ^~~~~~~~~~~
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    
    ----------------------------------------

```

I am running Manjaro Linux 18.0.0 (a derivative of Arch Linux) and I am running Python 3.7.1. Are there any additional packages I must install?

**Comment by nya3jp** [comment]
Hi!

On Linux prebuilts are used usually so you don't need to built the libraries by yourself, but prebuilt for Python 3.7 was not available. I've just built and uploaded it, so would you mind retrying installation?

BTW, the procedure to build the library by yourself is described in README, but it's written in Japanese, sorry :(

**Comment by kir12** [comment]
Hi again, sorry for the late reply! As it turns out, your uploading the Python 3.7 prebuilt did the trick. I did, however, encounter an entourage of other problems that delayed my responding to this Issue.

Thanks a lot for maintaining this project! Believe it or not it's actually quite useful.

**Comment by mudit67** [comment]
Hey!

I am facing the same issue while installing the library.

I am using ubuntu 20.04 and python3.8.10

Error message:
animeface/_nvxs.c:21:10: fatal error: nv_core.h: No such file or directory
         21 | #include "nv_core.h"
            |          ^~~~~~~~~~~
      compilation terminated.
      error: command 'x86_64-linux-gnu-gcc' failed with exit status 1

What might be wrong here?

**Comment by nya3jp** [comment]
Hi,

I've just released v2.0.0 and now wheels (prebuilt binary packages) are available for Python 3.6--3.10 on Linux and macOS.
